calculator.html

```diff
@@ -5,24 +5,28 @@
   <script src="./scripts/calculator.js"></script>
 </head>
 <body>
-  <div id="frame"></div>
-  <div id="calcBody"></div>
-  <div id="screen"></div>
-  <div><button id="button1" class="buttonText">All clear</button></div>
-  <div><button id="button2" class="button"><p class="buttonText">/</p></button></div>
-  <div><button id="button3" class="button"><p class="buttonText">7</p></button></div>
-  <div><button id="button4" class="button"><p class="buttonText">8</p></button></div>
-  <div><button id="button5" class="button"><p class="buttonText">9</p></button></div>
-  <div><button id="button6" class="button"><p class="buttonText">*</p></button></div>
-  <div><button id="button7" class="button"><p class="buttonText">4</p></button></div>
-  <div><button id="button8" class="button"><p class="buttonText">5</p></button></div>
-  <div><button id="button9" class="button"><p class="buttonText">6</p></button></div>
-  <div><button id="button10" class="button"><p class="buttonText">-</p></button></div>
-  <div><button id="button11" class="button"><p class="buttonText">1</p></button></div>
-  <div><button id="button12"class="button"><p class="buttonText">2</p></button></div>
-  <div><button id="button13" class="button"><p class="buttonText">3</p></button></div>
-  <div><button id="button14" class="button"><p class="buttonText">+</p></button></div>
-  <div><button id="button15"><p class="buttonText">0</p></button></div>
-  <div><button id="button16" class="button"><p class="buttonText">.</p></button></div>
-  <div><button id="button17" class="button"><p class="buttonText">=</p></button></div>
+  <div id="frame">
+    <div id="calcBody">
+      <div id="screen"></div>
+      <div class="calc-row">
```

Comment:
This "calc-row" element will spread whatever is placed inside of it along the width of the calculator
<img width="573" alt="Screen Shot 2022-04-24 at 11 49 28 AM" src="https://user-images.githubusercontent.com/13542922/164991775-c2281877-0bef-4e13-90fb-990619649b5d.png">
.